src/components/AccountModule/connection-hooks.js

```diff
@@ -63,13 +63,26 @@ export function useWalletConnectionDetails(
 ) {
   const { walletNetworkName, hasNetworkMismatch } = useNetworkConnectionData()
   const theme = useTheme()
-  let connectionDetails = {}
   const isWalletAndClientSynced =
     Math.abs(walletSyncDelay - clientSyncDelay) <= OK_PROVIDER_SYNC_DELAY
   const networkSlowdown =
     walletSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     clientSyncDelay >= MILD_PROVIDER_SYNC_DELAY &&
     isWalletAndClientSynced
+  const networkName = network.shortName.toLowerCase()
+
+  let connectionDetails = {
+    connectionMessage: `Connected to ${walletNetworkName}`,
+    connectionMessageLong: `Connected to Ethereum ${walletNetworkName} Network`,
+    connectionColor: theme.positive,
+  }
+
+  if (
+    clientListening &&
```

Comment:
We _still_ need to check the native client connection; if not we could have a false positive when the client websockets suddenly close.